incubator/tensorflow-inception/values.yaml

```diff
@@ -0,0 +1,35 @@
+## TensorFlow Serving server image version
+## ref: https://hub.docker.com/r/bitnami/tensorflow-serving/tags/
+##
+server:
+  image: tompizmor/tensorflow-serving:latest
```

Comment:
Did you mean to use `bitnami/tensorflow-serving` instead? Also it's not recommended to use images tagged `latest` in charts. It does not follow best practices